tools/cache_utils.py

```diff
@@ -12,6 +12,8 @@
 from sheerwater_benchmarking.utils.secrets import postgres_write_password
 from sheerwater_benchmarking.utils.caching import read_from_postgres
 
+POSTGRES_IP = "postgres.shared.rhizaresearch.org"
```

Comment:
using the new postgres url, but its also switching from using the service name or a hard coded ip address to using the externally routable domain name and unifying all the calls to use this one name instead of either the internal name or ip depending on call site. Not sure if that is the right thing to use or if it should be using the internal service name.